Sources/ElasticSwift/Requests/DeleteRequest.swift

```diff
@@ -11,20 +11,24 @@ import Foundation
 public class DeleteRequestBuilder: RequestBuilder {
     
     typealias BuilderClosure = (DeleteRequestBuilder) -> Void
+
+    public typealias RequestType = DeleteRequest
     
     let client: ESClient
-    var index: String?
+    var index: String
     var type: String?
-    var id: String?
+    var id: String
     var version: Int?
-    var completionHandler: ((DeleteResponse?, Error?) -> Void)?
+    public var completionHandler: ((DeleteResponse?, Error?) -> ())?
```

Comment:
Sorry, I've just realized that you asked me to use Void.
In fact there is no difference between Void and ()
```typealias Void = ()```
it's a styling choice ... I prefer () because there are less char to type ;)